Header header
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
float32 confidence
bool hand_detected
